AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContextDelegate.h

```diff
@@ -17,15 +17,15 @@ NS_ASSUME_NONNULL_BEGIN
  * @param authTokenContext The auth token context.
  * @param authToken The auth token.
  */
-- (void)authTokenContext:(MSAuthTokenContext *)authTokenContext didSetNewAuthToken:(nullable NSString *)authToken;
+- (void)authTokenContext:(MSAuthTokenContext *)authTokenContext didSetAuthToken:(nullable NSString *)authToken;
 
 /**
  * A callback that is called when a new user signs in.
  *
  * @param authTokenContext The auth token context.
  * @param authToken The auth token.
  */
-- (void)authTokenContext:(MSAuthTokenContext *)authTokenContext didSetNewAccountIdWithAuthToken:(nullable NSString *)authToken;
+- (void)authTokenContext:(MSAuthTokenContext *)authTokenContext didUpdateAccountIdWithAuthToken:(nullable NSString *)authToken;
```

Comment:
Should we pass the account id as a parameter here?

Reply:
We do. There will be a separate work item for this one.